builder/bundleset.go

```diff
@@ -221,6 +221,49 @@ func validateBundle(b *bundle) error {
 	return nil
 }
 
+// parsePundleFile parses a file at filename containing a flat list of package
+// names to be converted to one-package bundles (pundles). Returns a list of
+// pointers to bundle objects created from this list.
+func parsePackageBundleFile(filename string) (map[string]bool, error) {
+	packages := make(map[string]bool)
+	contents, err := ioutil.ReadFile(filename)
+	if err != nil {
+		return nil, err
+	}
+
+	pundleList := strings.Split(string(contents), "\n")
+	for _, p := range pundleList {
+		if len(p) == 0 {
+			continue
+		}
+
```

Comment:
Is it worth adding 'if p[0] == '#' { continue ; }' so you can have commented out pundles in your pundle file, or even real comments?

Reply:
Yep, done.